robot/rospackages/src/astro_arm/package.xml

```diff
@@ -4,17 +4,15 @@
   <description>
     <p>URDF Description package for astro_arm</p>
     <p>This package contains configuration data, 3D models and launch files
-for ASTRO</p>
+for the ASTRo Manipulator</p>
   </description>
-  <author>TODO</author>
-  <maintainer email="TODO@email.com" />
+  <author>Space Concordia Robotics</author>
+  <maintainer email="robotics@spaceconcordia.ca" />

```

Comment:
oh thats clever, i was putting my own contact info but like this people will always have someone to reach. then again it may not be the person they need to reach